### PR TITLE
additional feature suggest and order history part 1

### PR DESCRIPTION
**Pull request by toilaloc**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/xxxx)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by toilaloc** [comment]
ready

File: app/Http/Controllers/Client/OrderController.php [hunk 1]
```diff
@@ -94,4 +94,23 @@ public function orderSuccess()
...
     }
+
+    public function confirmOrder($id)
+    {
+        if (Auth::check()) {
+            $foundOrder = $this->orderRepo->findOrFail($id);
+            if (Auth::id() == $foundOrder->user_id) {
+                $confirmOrder = $this->orderRepo->update(
+                    $id,
+                    ['status' => 5]
```
Comment (quoctd-0607) [inline app/Http/Controllers/Client/OrderController.php]: status 5 là gì em?

File: app/Providers/AppServiceProvider.php [hunk 2]
```diff
@@ -55,5 +57,12 @@ public function boot()
     {
         Blade::withoutDoubleEncoding();
         Paginator::useBootstrap();
+        view()->composer(
+            '*',
+            function ($view) {
+                $categoriesMenu = Category::whereNull('parent_id')->take(7)->get();
```
Comment (quoctd-0607) [inline app/Providers/AppServiceProvider.php]: config 7 nhé

File: app/Http/Controllers/Client/OrderController.php (hunk 1 above)
Comment (toilaloc) [reply app/Http/Controllers/Client/OrderController.php]: dạ để em add vào config

**Comment by toilaloc** [comment]
ready

**Pull request merged**